tests/test_human_activities.py

```diff
@@ -0,0 +1,185 @@
+import hashlib
+import os
+import random
+import sys
+import string
+import unittest
+
+from PySide2.QtTest import QTest
+from PySide2.QtCore import Qt
+from PySide2.QtWidgets import QGraphicsScene, QGraphicsView
+
+import angr
+from angrmanagement.ui.main_window import MainWindow
+from angrmanagement.ui.dialogs.rename_label import RenameLabel
+from angrmanagement.ui.dialogs.rename_node import RenameNode
+
+from common import setUp, test_location
+
+from slacrs import Slacrs
+from slacrs.model import HumanActivityVariableRename, HumanActivityFunctionRename, HumanActivityClickBlock, HumanActivityClickInsn
+
+
+class TestHumanActivities(unittest.TestCase):
+    def setUp(self):
+        setUp()
+        # set up a random database through environment variable SLACRS_DATABASE
+        self.db_name = ''.join([random.choice(string.ascii_letters) for _ in range(8)])
+        os.environ['SLACRS_DATABASE'] = f"sqlite:////tmp/{self.db_name}.sqlite"
```

Comment:
Nitpick: Let's use an in-memory sqlite database instead.

Note to myself: angr management won't respect `SLACRS_DATABASE` in the future. We should always use the CHECRS-backend-selection plugin.